boards/arm/imx6/sabre-6quad/scripts/Make.defs

```diff
@@ -34,7 +34,6 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCPUFLAGS = -mcpu=cortex-a9 -mfpu=vfpv4-d16 -mthumb
```

Comment:
Is `-mthumb` flag now passed elsewhere?

Reply:
I's only passed to imx6 by de95a8550f8e468396545eab1be11fd33b12be97